var/spack/tests/compilers/fortran/hello.f95

```diff
@@ -0,0 +1,6 @@
+      program line
+
+      write (*,*) "Hello world from FORTRAN"
+      write (*,*) "YES!"
+
+      end
```

Comment:
Could get rid of the spacing before `program` and `end` (or `end program line`).